core/src/templates/components/card/card.twig

```diff
@@ -23,14 +23,14 @@
  * - card_button_label: The text label of the button component.
  */
 #}
-<div{{ attributes }} class="su-card {{ modifier_class }}">
+<article{{ attributes }} class="su-card {{ modifier_class }}">
```

Comment:
> The article element represents a component of a page that consists of a self-contained composition in a document, page, application, or site and that is intended to be independently distributable or reusable, e.g. in syndication. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.

> W3C Specification: http://www.w3.org/TR/html5/semantics.html#the-article-element

I would agree then that the card should/can be an article.

Reply:
From http://html5doctor.com/the-article-element/

> The smell test for going independent
> An independent piece of content, one suitable for putting in an <article> element, is content that makes sense on its own.

Reply:
I'm ok with the card being an article as well :)

Reply:
As this is the wrapper tag, we may have to wait until version 6 to get this out. This would break the 5.x API. I'll cut a new v6 branch and point this PR to it.